### PR TITLE
Update hostname_tagging.md

### PR DESCRIPTION
**Pull request by AxelleFesard**
Add host and containers custom tagging options

### What does this PR do? What is the motivation?

Add host and containers custom tagging options for OTEL prefixing resource attributes with datadog.\<key\>.tag.
Also presents the host tagging option in the collector.

### Merge instructions

- [X] Please merge after reviewing

### CODE REVIEW

**Review by mx-psi — COMMENTED** [review]
Thanks!

**Review by michaelcretzman — APPROVED** [review]

**Comment by michaelcretzman** [comment]
@AxelleFesard once you have incorporated @mx-psi suggestions, please let Doc know of the #documentation channel and we can merge this PR.

File: content/en/opentelemetry/collector_exporter/hostname_tagging.md [hunk 1]
```diff
@@ -427,8 +427,58 @@ Attributes:
...
+      ##
+      ## To attach tags to telemetry signals regardless of the host, use a processor instead.
+      #
+      tags: ["team:infra", "<TAG_KEY>:<TAG_VALUE>"]
+```
+See all configurations options [here][6].
+
+#### In OTEL Instrumentation
+
+Custom host tags can also be set in your OTEL instrumentation by prefixing resource attributes by `datadog.host.tag`.
```
[the remark this answers is not in the document]
Comment (AxelleFesard) [reply content/en/opentelemetry/collector_exporter/hostname_tagging.md]: Done!

**Review by mx-psi — APPROVED** [review]
Thanks for the PR!!

**Review by mackjmr — APPROVED** [review]
LGTM, thanks for adding this! 🚀

**Review by aliciascott — APPROVED** [review]

**Pull request merged**